commands/security.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types = 1);
+
+use Kirby\CLI\CLI;
+use Kirby\Http\Remote;
+use Kirby\Http\Url;
+use Kirby\Toolkit\I18n;
+
+return [
+	'description' => 'Performs security checks of the site',
+	'command' => static function (CLI $cli): void {
+		$kirby        = $cli->kirby();
+		$system       = $kirby->system();
+		$updateStatus = $system->updateStatus();
+		$messages     = [
+			...array_column($updateStatus?->messages() ?? [], 'text'),
+			...$updateStatus->exceptionMessages()
+		];
+
+		if ($kirby->option('debug', false) === true) {
+			$messages[] = I18n::translate('system.issues.debug');
+		}
+
+		if ($kirby->environment()->https() !== true) {
+			$messages[] = I18n::translate('system.issues.https');
+		}
+
+		// checks exposable urls of the site
+		// works only site url is absolute since can't get it in CLI mode
+		// and CURL won't work for relative urls
+		if (Url::isAbsolute($kirby->url())) {
```

Comment:
If the URL isn't absolute, these checks will be skipped and this could be unnoticed by the user. Maybe we could add an `else` case that adds an error message like "Could not check for exposed folders as the site URL is not absolute".